src/screens/SSICredentialsRequiredScreen/index.tsx

```diff
@@ -151,24 +161,30 @@ const SSICredentialsRequiredScreen: FC<Props> = (props: Props): JSX.Element => {
         ? undefined
         : () => onItemPress(itemInfo.item.id, availableCredentials.get(itemInfo.item.id)!, itemInfo.item.purpose);
 
+    const checkIsMatching = (itemInfo: any, selectedCredentials: any, pex: any, getOriginalVerifiableCredential: any) => {
+      if (!selectedCredentials.has(itemInfo.item.id)) {
+        return false;
+      }
+
+      const credentials = selectedCredentials.get(itemInfo.item.id)!.map((uniqueVC: any) => getOriginalVerifiableCredential(uniqueVC.verifiableCredential));
```

Comment:
Don't introduce any

Reply:
That was bad yes. Limited to GPT 3.5 atm and forgot to check what it wrote.